vue-frontend/src/components/home-new/HomeIndex.vue

```diff
@@ -30,8 +30,22 @@ import CTA from "@/components/home-new/CTASection.vue";
 import Blog from "@/components/home-new/BlogSection.vue";
 import Contribute from "@/components/home-new/ContributeSection.vue";
 import ScreenFooter2 from "@/components/partials/ScreenFooter2.vue";
+import Config from "@/config.js";
+import { useMeta } from "vue-meta";
 
 export default {
+  setup() {
```

Comment:
Can't we add it in `pages/index.vue?`, as for both new and old home page meta data is same?